sos/report/plugins/ovn_central.py

```diff
@@ -86,10 +88,8 @@ def get_tables_from_schema(self, filename, skip=None):
 
     def add_database_output(self, tables, cmds, ovn_cmd):
```

Comment:
`cmds` argument is redundant now so we shall drop it..?

Reply:
You actually found a bug, that parameter was going nowhere. This fixes it:

```diff
$ git diff 
diff --git a/sos/report/plugins/ovn_central.py b/sos/report/plugins/ovn_central.py
index 389fb13a..28aa4415 100644
--- a/sos/report/plugins/ovn_central.py
+++ b/sos/report/plugins/ovn_central.py
@@ -86,7 +86,7 @@ class OVNCentral(Plugin):
             self._log_error(f"DB schema {filename} has no 'tables' key")
         return None
 
-    def add_database_output(self, tables, cmds, ovn_cmd):
+    def add_database_output(self, tables, ovn_cmd):
         """ Collect OVN database output """
         if tables:
             return [f"{ovn_cmd} list {table}" for table in tables]
@@ -138,6 +138,7 @@ class OVNCentral(Plugin):
         self.add_cmd_output(appctl_cmds, foreground=True,
                             container=self.container_name, timeout=30)
 
+        # MicroOVN currently does not support this
         if not pfx:
             dfl = "debug/chassis-features-list"
             self.add_cmd_output(f"{pfx}ovn-appctl -t {northd_socket} {dfl}",
@@ -152,6 +153,9 @@ class OVNCentral(Plugin):
             f"{pfx}ovn-nbctl {nolo} get-connection",
         ]
 
+        self.add_cmd_output(nbctl_cmds, foreground=True,
+                            container=self.container_name, timeout=30)
+
         sbctl_cmds = [
             f"{pfx}ovn-sbctl {nolo} show",
             f"{pfx}ovn-sbctl {nolo} lflow-list",
@@ -159,13 +163,16 @@ class OVNCentral(Plugin):
             f"{pfx}ovn-sbctl {nolo} get-connection",
         ]
 
+        self.add_cmd_output(sbctl_cmds, foreground=True,
+                            container=self.container_name, timeout=30)
+
         # backward compatibility
         for path in ['/usr/share/openvswitch', '/usr/share/ovn',
                      '/snap/microovn/current/share/ovn']:
             if self.path_exists(self.path_join(path, 'ovn-nb.ovsschema')):
                 nb_tables = self.get_tables_from_schema(self.path_join(
                     path, 'ovn-nb.ovsschema'))
-                cmds.extend(self.add_database_output(nb_tables, nbctl_cmds,
+                cmds.extend(self.add_database_output(nb_tables,
                                                      f"{pfx}ovn-nbctl {nolo}"))
 
         for path in ['/usr/share/openvswitch', '/usr/share/ovn',
@@ -173,7 +180,7 @@ class OVNCentral(Plugin):
             if self.path_exists(self.path_join(path, 'ovn-sb.ovsschema')):
                 sb_tables = self.get_tables_from_schema(self.path_join(
                     path, 'ovn-sb.ovsschema'), ['Logical_Flow'])
-                cmds.extend(self.add_database_output(sb_tables, sbctl_cmds,
+                cmds.extend(self.add_database_output(sb_tables,
                                                      f"{pfx}ovn-sbctl {nolo}"))
 
         # If OVN is containerized, we need to run the above commands inside
```

Reply:
This has been implemented and pushed.